lms/templates/static_templates/honor.html

```diff
@@ -10,7 +10,7 @@
         <div class="row">
           <div class="col-md-9">
             ## include privacy policy from git repo
-            ${urllib2.urlopen('https://gymnasium.github.io/static-site-content/privacy-policy.html').read()}
+            ${gymcms.render('faq')}
```

Comment:
Did you mean to render the FAQ page here? Would the Privacy Policy page be better?

Reply:
This should have been the privacy policy - likely a dumb c/p job.  We don't actually link to an honor page anywhere on the site, handily, but it's good to do it to be thorough. I'll submit this change too.